tests/Test.ADAL.NET.Unit/Test.ADAL.NET.Unit.csproj

```diff
@@ -59,6 +59,10 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Data" />
+    <Reference Include="System.IdentityModel" />
+    <Reference Include="System.IdentityModel.Tokens.Jwt, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
why are we referencing JwtTokenHandler for the test project? More specifically, what is being tested?